flink/v1.18/flink/src/test/java/org/apache/iceberg/flink/TestFlinkCatalogDatabase.java

```diff
@@ -29,88 +29,93 @@
 import org.apache.iceberg.catalog.TableIdentifier;
 import org.apache.iceberg.types.Types;
 import org.assertj.core.api.Assertions;
-import org.junit.After;
-import org.junit.Assert;
-import org.junit.Assume;
-import org.junit.Test;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.TestTemplate;
 
-public class TestFlinkCatalogDatabase extends FlinkCatalogTestBase {
+public class TestFlinkCatalogDatabase extends CatalogTestBase {
 
-  public TestFlinkCatalogDatabase(String catalogName, Namespace baseNamespace) {
-    super(catalogName, baseNamespace);
-  }
-
-  @After
+  @AfterEach
   @Override
   public void clean() {
     sql("DROP TABLE IF EXISTS %s.tl", flinkDatabase);
     sql("DROP DATABASE IF EXISTS %s", flinkDatabase);
     super.clean();
   }
 
-  @Test
+  @TestTemplate
   public void testCreateNamespace() {
-    Assert.assertFalse(
-        "Database should not already exist",
-        validationNamespaceCatalog.namespaceExists(icebergNamespace));
-
+    Assertions.assertThat(validationNamespaceCatalog.namespaceExists(icebergNamespace))
+        .as("Database should not already exist")
+        .isFalse();
     sql("CREATE DATABASE %s", flinkDatabase);
 
-    Assert.assertTrue(
-        "Database should exist", validationNamespaceCatalog.namespaceExists(icebergNamespace));
+    Assertions.assertThat(validationNamespaceCatalog.namespaceExists(icebergNamespace))
+        .as("Database should exist")
+        .isTrue();
 
     sql("CREATE DATABASE IF NOT EXISTS %s", flinkDatabase);
-    Assert.assertTrue(
-        "Database should still exist",
-        validationNamespaceCatalog.namespaceExists(icebergNamespace));
+
```

Comment:
unnecessary newline here and in a bunch of places below

Reply:
ack